ci_scripts/ci_post_clone.sh

```diff
@@ -31,7 +31,9 @@ APP_ID_IOS = org.mozilla.ios.FirefoxVPN
 NETEXT_ID_IOS = org.mozilla.ios.FirefoxVPN.network-extension
 EOF
 
-curl -L https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/mozillavpn.v2.mozillavpn.cache.level-3.toolchains.v3.qt-ios.latest/artifacts/public%2Fbuild%2Fqt6_ios.zip --output qt_ios.zip
+# THIS IS A QT HOTFIX, TO CONTINUE USING 6.2.4 - Change back to curling `latest` when we move to Qt 6.5.1.
+# curl -L https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/mozillavpn.v2.mozillavpn.cache.level-3.toolchains.v3.qt-ios.latest/artifacts/public%2Fbuild%2Fqt6_ios.zip --output qt_ios.zip
+curl -L https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/c9a9Uw_7Tm2L2T_QU5_Ydg/runs/0/artifacts/public%2Fbuild%2Fqt6_ios.zip --output qt_ios.zip
```

Comment:
Will this affect non-Mozilla developers? Will they be able to access this link?

Reply:
`ci_post_clone.sh` is only used by Xcode Cloud - it's not used in any other workflow, to the best of my knowledge. I'm not quite sure how Xcode Cloud has permission to hit this, but it apparently already did (see the commented out line), so we're just going with it. :)